src/app/services/reports.service.ts

```diff
@@ -19,7 +19,7 @@ export class ReportsService {
   private readonly HEADERS = new HttpHeaders({ 'Content-Type': 'application/json' });
   
   /** Milliseconds per year */
-  private readonly MILLIS_PER_YEAR: number = 3.154e+10;
+  readonly MILLIS_PER_YEAR: number = 3.154e+10;
```

Comment:
Any reason why this needs to be non-private?

Reply:
Yes, it needed to be accessed for a few tests.